services/app-web/src/components/ChangeHistory/ChangeHistoryV2.tsx

```diff
@@ -28,35 +28,54 @@ export const ChangeHistoryV2 = ({
 
```

Comment:
We're not performing the above filter in the SubmissionRevisionSummary component so our indices won't line up. I forget why we decided to filter these out in the first place? Either way the two should match

Reply:
It came up in this PR comment that we don't want to pull unlock info from rates https://github.com/Enterprise-CMCS/managed-care-review/pull/2350#discussion_r1571208151. I updated the SubmissionRevisionSummary to also have this filter